autoscale/src/test/java/com/sequenceiq/periscope/InjectTest.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.periscope;
+
+import org.junit.Test;
```

Comment:
I suggest using JUnit 5 instead (also in all the analogous tests of other apps).

Reply:
please *do* use JUnit5